store/conf/attrs/zimbra-attrs.xml

```diff
@@ -9563,4 +9563,19 @@ TODO: delete them permanently from here
   <defaultCOSValue>Received,DKIM-Signature,Authentication-Results,Received-SPF,Message-ID,Content-Type,Content-Disposition,Content-Transfer-Encoding,MIME-Version,Auto-Submitted</defaultCOSValue>
   <desc>Comma separated list of sieve immutable headers</desc>
 </attr>
+
+<attr id="3020" name="zimbraFeatureSearchHistoryEnabled" type="boolean" cardinality="single" optionalIn="account,cos" flags="domainAdminModifiable,accountInfo,accountInherited" since="8.8.1">
+  <desc>Whether to enable Search History</desc>
+  <defaultCOSValue>TRUE</defaultCOSValue>
+</attr>
+
+<attr id="3021" name="zimbraNumSearchesForSavedSearchPrompt" type="integer" min="0" cardinality="single" optionalIn="account,cos" flags="accountInherited" since="8.8.1">
+  <desc>Searching for the same query this many times will result in a prompt offering the user to save this query as a search folder. This will only happen once per query. A value of 0 disables this feature.</desc>
+  <defaultCOSValue>3</defaultCOSValue>
+</attr>
+
+<attr id="3022" name="zimbraSearchHistoryDuration" type="duration" cardinality="single" optionalIn="account,cos" flags="accountInherited" since="8.8.1">
```

Comment:
minor: perhaps "Age" instead of "Duration"?

Reply:
yeah that would be more clear, especially because the parameter name used by the api is "maxAgeMillis"